jenkins/runlocal

```diff
@@ -21,6 +24,7 @@ make -C server/rpm clean rpm
 # $ buildah login -u='$app' -p="${__LOGIN_SECRET__}" ${PB_CONTAINER_REG}
 
 RPM_PATH=${HOME}/rpmbuild/RPMS/noarch/pbench-server-*.rpm bash -ex ./server/pbenchinacan/container-build-ci.sh
+buildah push localhost/pbench-server:${PB_SERVER_IMAGE_TAG} ${PB_CONTAINER_REG}/${PB_ORG_NAME}/pbench-server:${PB_SERVER_IMAGE_TAG}
 buildah push localhost/${PB_SERVER_IMAGE_NAME}:${PB_SERVER_IMAGE_TAG} ${PB_CONTAINER_REG}/${PB_ORG_NAME}/${PB_SERVER_IMAGE_NAME}:${PB_SERVER_IMAGE_TAG}
```

Comment:
Why is the repository for the "stand-alone" image hard-coded to `pbench-server` when the repository for the canned image is parameterized (and now specified as `pbench-server-local`)?

I think I would prefer that both images used the same repository (with suitable tags), but, if we really want them in separate repos then it seems like they should both get similar treatment here (and, I'm inclined toward either hard-coding them or pulling the definition from some place common).

Reply:
> Why is the repository for the "stand-alone" image hard-coded to `pbench-server` when the repository for the canned image is parameterized (and now specified as `pbench-server-local`)?

Because the image we are going to publish to run in staging and production is always the one named `pbench-server` as build by `container-build-sa.sh`.

> I think I would prefer that both images used the same repository (with suitable tags), but, if we really want them in separate repos then it seems like they should both get similar treatment here (and, I'm inclined toward either hard-coding them or pulling the definition from some place common).

I would agree if they were the same image but they are not.  The image built by `container-build-sa.sh` is does not include any setup of an internal `/srv/pbench` directory.

The purpose of the "sa" image is to be run in staging and production.

The purpose of the "ci" image is to be run in the CI environment without any external dependencies.

Now, we could consider a slightly different way of running the "in-a-can" environment where we only use the "sa" image, and all the internal container setup is performed at runtime for the CI.

But that is beyond the scope of this PR.

Reply:
> we could consider a slightly different way of running the "in-a-can" environment where we only use the "sa" image, and all the internal container setup is performed at runtime for the CI.

That would be trading the "can-ed-ness" of the canned server for a wrapper around the SA image, which seems like six-of-one vs. a half-dozen of the other.  And, personally, I like the canned server approach, although we've had to wrap that with a script now, so it's possible that its advantages have been lost.